todo_app/data/trello_items.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+import requests
+import os
+
+API_URL = "https://api.trello.com"
+BOARD_ID = os.getenv('BOARD_ID')
+BASE_PAYLOAD = {
+    'key': os.getenv('API_KEY'),
+    'token': os.getenv('TOKEN')
+}
+class Status(Enum):
+    TO_DO = "To Do"
+    DONE = "Done"
+
+class Item:
+    def __init__(self, id, title, status = Status.TO_DO.value):
+        self.id = id
+        self.title = title
+        self.status = status
+    
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
```

Comment:
I'd be inclined to pull this into its own file personally, `from_trello_card` is obviously related but conceptually we want this to represent items regardless of their source